### DISCUSSION TITLE
A small question about JS

### DISCUSSION THREAD

**Discussion by dara-marotto** [community/community#43598] [Other Feature Feedback, Questions, & Ideas] [answered]
Why I dont need to add `#` to get the element I want in the `getElementByID`?
I know in `querySelector` I need to put `#` for ID's.

**Comment by kelwin-hollunder** [comment, marked as answer]
In `getElementByID` the method is already waiting an ID as an argument.
In `querySelector` he awaits a class or ID, because of this you need to specify with #.
